### PR TITLE
Coachmark/TeachingBubble: Fix SCSS selectors for buttons and Close Icon

### LINKED ISSUE #4812: Teaching Bubble close icon on hover
#### Describe the issue:

The close icon on the teaching bubble turns white on hover instead of remaining blue. This is reproducible on the office fabric doc page. <a>https://developer.microsoft.com/en-us/fabric#/components/teachingbubble</a>

### PR DESCRIPTION
**Pull request by leddie24**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #4812 
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

Updates SCSS selectors to properly show teaching bubble close icon and primary/secondary button styles.

**Old**:
old-coachmark

**New**:
new-coachmark

I'm working on a separate PR for a lot of positioning fixes with Coachmark/TeachingBubble that will fix the cut off.  It will be a big PR, so I wanted to separate this small style change in this PR first.

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by JasonGore** [comment]
Can you add a basic Coachmark snapshot test (from master before your PR), switch to your branch (with new test and snapshot present), then confirm the differences in the snapshot introduced by your PR make sense? Thanks

File: packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.scss [hunk 1]
```diff
@@ -155,7 +159,7 @@
 }
 
 //Primary button style override
-.root .primaryButton {
+.primaryButton {
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.scss]: removing specificity here could cause issues.

Comment (micahgodbolt) [reply packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.scss]: agreed...i'm guessing these styles are overriding previous styles...probably applied via a classname, so any change in load order could break this.

Comment (micahgodbolt) [reply packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.scss]: If you want to simplify this, you could pass the styles into the button styles prop

**Pull request merged**